SHIELD-GLOSSARY.md

```diff
@@ -0,0 +1,95 @@
+# Shield Glossary
+
+## Introduction
+
+In the greater [Shield project](https://wiki.mozilla.org/Firefox/Shield),
+with the code projects [Normandy](https://github.com/mozilla/normandy) and
+[Delivery Console](https://github.com/mozilla/delivery-console/) in mind,
+there are a lot of different terms to remember and understand.
+
+Here we try
+to list them with the goal of...:
+
+1. Getting everyone to use the same terms
+
+2. Understanding what they are and how they're different to each other
+
+3. Make it easier to people new to the using the projects and people
+   coding the code for the services
+
+
+
+## Glossary
+
+* **Addon Studies**
+
+    * When you first install an extension, then a recipe that sends the data to supply the extension to do a study.
+    * *Alias:* Opt Out Studies
+
+* **Delivery Console**
+
+    * Web UI interface for administering the content in Normandy.
+    * https://github.com/mozilla/delivery-console/
+    * *Alias:* DC
+
+* **Feature Flagging**
+
+    * A project to improve the functionality of overriding and making preferences available at all for new features.
+    * The objective is to make adding and flipping preferences safer.
+    * *Alias:* Feature Gating
+
+* **Filter Expression**
+
+    * A JEXL string stored in Normandy, as part of recipes, that the Normandy Client reads to determine whether a certain client should invoke the action.
+    * *Alias:* JEXL, MozJEXL
```

Comment:
I'm not sure I would list JEXL and MozJEXL as aliases of Filter Expression, especially since you use JEXL in the definition. I would separate these: JEXL is a language, and a filter expression is something written in that language.